samples/outbox/sqltransport-nhpersistence-ef/SqlServer_3/Receiver/OrderSubmittedHandler.cs

```diff
@@ -17,7 +19,7 @@ public class OrderSubmittedHandler :
 
         var storageContext = context.SynchronizedStorageSession.Session();
 
-        var dbConnection = storageContext.Connection;
+        var dbConnection = (SqlConnection) storageContext.Connection;
         using (var receiverDataContext = new ReceiverDataContext(dbConnection))
         {
             var dbTransaction = ExtractTransactionFromSession(storageContext);
```

Comment:
@SzymonPobiega Can you answer this:

> why not use `storageContext.Transaction`?

I have no idea what is the purpose of creating a dummy command and enlisting it.